docker-compose.yml

```diff
@@ -33,8 +33,10 @@ services:
         ports:
             - 6443:443
             - 6444:6444
+        sysctls:
+            - net.ipv4.ip_unprivileged_port_start=0
```

Comment:
Maybe should be changed to 23 so ssh port isnt exposed?

Reply:
This isn't which ports are exposed, it's which ports a non-user is allowed to listen on.

Docker itself [includes](https://github.com/moby/moby/pull/41030/files#diff-9f91bff23e0bd70d6429b63d9db2d8180d2e89cdb64db4fb3e10a96f74d36271R791-R793) `net.ipv4.ip_unprivileged_port_start = 0`, just not containerd yet.